curation-migration-datasync/helpers/timeConverter.ts

```diff
@@ -0,0 +1,51 @@
+import { ScheduledSurfaceGuid } from '../dynamodb/types';
+
+/**
+ * The given scheduled date is converted to 3am in their local time
+ * for the corresponding NewTab, and then converted to unix epoc timestamp.
+ *
+ * For e.g scheduledDate `2022-06-01` is converted to:
+ * istTimestamp = '2022-05-31 00:21:30'; // subtract one day and set to 930 pm
+ * berlinTime = '2022-06-01 00:01:00'; //add one hour
+ * ukTime = '2022-06-01 00:03:00'; // add 3 hours with date
+ * estTime = '2022-06-01 00:07:00'; //add 7 hrs with date
```

Comment:
Nitpick: "with date" should be replaced with "to the timestamp provided" or similar.

Reply:
the function returns the `timestamp` and gets the date as input..so we are adding/subtracting hrs to the Date object and converting it to epoch timestamp, updated the function doc to make it less confusing